[2023-11-08T14:56:05.667+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yaCIss.test manual__2023-11-08T14:56:03.098492+00:00 [queued]>
[2023-11-08T14:56:05.676+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yaCIss.test manual__2023-11-08T14:56:03.098492+00:00 [queued]>
[2023-11-08T14:56:05.677+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-08T14:56:05.693+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): test> on 2023-11-08 14:56:03.098492+00:00
[2023-11-08T14:56:05.699+0000] {standard_task_runner.py:57} INFO - Started process 90 to run task
[2023-11-08T14:56:05.702+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'yaCIss', 'test', 'manual__2023-11-08T14:56:03.098492+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/data/yarabi.py', '--cfg-path', '/tmp/tmpqbh8ptvt']
[2023-11-08T14:56:05.706+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask test
[2023-11-08T14:56:05.763+0000] {task_command.py:416} INFO - Running <TaskInstance: yaCIss.test manual__2023-11-08T14:56:03.098492+00:00 [running]> on host a50cf6f085c6
[2023-11-08T14:56:05.843+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='yaCIss' AIRFLOW_CTX_TASK_ID='test' AIRFLOW_CTX_EXECUTION_DATE='2023-11-08T14:56:03.098492+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-08T14:56:03.098492+00:00'
[2023-11-08T14:56:05.845+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-08T14:56:05.846+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl --keepalive-time 600 -o ${AIRFLOW_HOME}/data/valeurs_foncieres.txt https://static.data.gouv.fr/resources/demandesde-valeurs-foncieres/20231010-093059/valeursfoncieres-2022.txt']
[2023-11-08T14:56:05.858+0000] {subprocess.py:86} INFO - Output:
[2023-11-08T14:56:05.869+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2023-11-08T14:56:05.869+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2023-11-08T14:56:05.970+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: Failed to create the file /opt/***/data/valeurs_foncieres.txt: No
[2023-11-08T14:56:05.972+0000] {subprocess.py:93} INFO - Warning: such file or directory
[2023-11-08T14:56:05.973+0000] {subprocess.py:93} INFO - 100   146  100   146    0     0   1445      0 --:--:-- --:--:-- --:--:--  1431
[2023-11-08T14:56:05.974+0000] {subprocess.py:93} INFO - curl: (23) Failure writing output to destination
[2023-11-08T14:56:05.975+0000] {subprocess.py:97} INFO - Command exited with return code 23
[2023-11-08T14:56:05.990+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 23.
[2023-11-08T14:56:05.995+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=yaCIss, task_id=test, execution_date=20231108T145603, start_date=20231108T145605, end_date=20231108T145605
[2023-11-08T14:56:06.008+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 4 for task test (Bash command failed. The command returned a non-zero exit code 23.; 90)
[2023-11-08T14:56:06.036+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-08T14:56:06.054+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
